doc/CONTRIBUTING.md

```diff
@@ -40,6 +40,27 @@ The mainline branch is `develop` (this will change post-1.0.0). Feature developm
 Name your feature branches with a name or handle of the branch owner as a prefix, followed by a brief description preceded by a slash, e.g. `cwgoes/eal-inference-bugfix`.
 
 Pushing minor changes (typo fixes) to another person's branch is fine. Ask before pushing major changes.
+## Version changelogs
+
+The format for the changelog follows the PVP format below.
+
+```
+  ︻ ︻ ︻ ︻
```

Comment:
I find this diagram pretty confusing, could we write it differently perhaps? Just list the sub-version-numbers.